c2cgeoform/templates/grid.jinja2

```diff
@@ -39,17 +39,20 @@
           refresh: '{{_('Refresh')}}',
           search: '{{_('Search')}}'
       },
-      converters: {
-          commands: {
-              from: function(value) {
-                  return '_id_';
-              },
-              to: function(value) {
-                  var delete_url = "{{request.route_url('c2cgeoform_action', id='_id_')}}".replace('_id_', value);
-                  return '' +
-                      '<a href="{{request.route_url('c2cgeoform_action', id='_id_')}}">{{_("Edit")}}</a>'.replace('_id_', value) +
-                      '<a href="#delete" onclick=confirmation("'+delete_url+'");>{{_("Delete")}}</a>';
-              },
+      formatters: {
+          "commands": function(column, row) {
+              var delete_url = "{{request.route_url('c2cgeoform_action', id='_id_')}}".replace('_id_', row._id_);
+              return '' +
+                  '<ul class="grid-row-actions">' +
+                  '  <li class="grid-row-action">' +
+                  '    <a class="btn-sm btn-primary glyphicon glyphicon-pencil" ' +
+                  '       title="{{_("Edit")}}" ' +
+                  '       href="{{request.route_url('c2cgeoform_action', id='_id_')}}"></a>'.replace('_id_', row._id_) +
+                  '  </li>' +
+                  '  <li class="grid-row-action">' +
+                  '    <a class="btn-sm btn-danger glyphicon glyphicon-remove" ' +
+                  '      title="{{_("Delete")}}" href="#delete" onclick=confirmation("'+delete_url+'");></a>' +
```

Comment:
I think that's better to use \ instance of + (language syntax instance of string concatenation)